### ISSUE TITLE
TypeHints.DeclareStrictTypes should disallow spaces after/before parenthesis

### ISSUE THREAD

**Issue by mhujer** [slevomat/coding-standard#200]
> Enforces having `declare(strict_types = 1)` at the top of each PHP file

Following are not detected by DeclareStrictTypes sniff (there are spaces after opening parenthesis and before closing parenthesis).

```php
declare( strict_types = 1 );
declare( strict_types = 1);
declare(strict_types = 1 );
```

**Comment by ondrejmirtes** [comment]
I think there’s a different sniff that enforces parenthesis and assign
operator spacing across all code, not just for declare.

**Comment by ondrejmirtes** [comment]
The README wording says that you should use strict_types, it’s not about
the exact look of the statement. Feel free to send a PR 😊

**Comment by mhujer** [comment]
I found that there is a `PSR2.ControlStructures.ControlStructureSpacing` but it does not handle `declare` statements yet. I will submit a PR directly to PHPCS.

Thanks for pointing me in the right direction @ondrejmirtes 👍
